charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -58,6 +58,8 @@ spec:
               value: {{ join "," .Values.tentacle.targetEnvironments | quote }}
             - name: "TargetRole"
               value: {{ join "," .Values.tentacle.targetRoles | quote }}
+            - name: "DefaultNamespace"
```

Comment:
We talk about renaming these properties to say "InitialXYZ" at one stage, did we not action it?

Reply:
Oh I don't remember this, but it makes sense to me and it makes it clearer that this might not be up-to-date with what's in server.

Is it better to do that in a separate PR? (I can change the default namespace one in this PR, and do the ther properties in a separate PR).

Reply:
Yeah separate PR is fine - or if you want to leave this one like this and rename them all together